.gitattributes

```diff
@@ -0,0 +1,8 @@
+/tests export-ignore
```

Comment:
Not sure. Same stuff in doctrine/doctrine2:
https://github.com/doctrine/doctrine2/blob/master/.gitattributes